source/npm/util.js

```diff
@@ -87,3 +91,14 @@ exports.verifyRecentNpmVersion = async () => {
 	const npmVersion = await exports.version();
 	verifyRequirementSatisfied('npm', npmVersion);
 };
+
+exports.checkIgnoreStrategy = ({files}) => {
+	const rootDir = pkgDir.sync();
+	const npmignoreExists = existsSync(resolve(rootDir, '.npmignore'));
```

Comment:
Destructuring fallacy. `resolve` is a generic meaningless name without context. Same with `existsSync`.

```suggestion
	const npmignoreExists = fs.existsSync(path.resolve(rootDir, '.npmignore'));
```

Reply:
Done